server/model/model.go

```diff
@@ -163,6 +163,6 @@ func (s *needStatus) Scan(src interface{}) error {
 	return nil
 }
 
-func (s needStatus) Value() (driver.Value, error) {
+func (s NeedStatus) Value() (driver.Value, error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method NeedStatus.Value should have comment or be unexported](https://app.codacy.com/app/coderockrdev/vitrine-social/pullRequest?prid=1995521)